intro/setup/index.html

```diff
@@ -0,0 +1,349 @@
+
+{% highlight python %}
+screen = pygame.display.set_mode(WIN_SIZE, flags=pygame.SCALED)
+{% endhighlight %}
+
+        There we go! Our window is much more fit for our desktop now. Notice the ratio of the window stays the same. 
+    </p>
+
+    <h3>Exiting the game</h3>
+    <p>
+        We have an interesting problem, if you tried closing the application with the X button, 
+        it won't work. This is because you haven't defined what you want pygame to do when the X button is 
+        pressed. The way you exit it as of now is to use ctrl + c in the terminal you are running it from.
+    </p>
+
+    <p>
+        Obviously, this isn't ideal, at all. Remember how we grabbed all the events 
+        that were going on in the window? That actually includes closing it as well. 
+        All we have to do is check if the window is being closed(i.e, the X button is being pressed),
+        and if it is we just need to exit our while loop. 
+
+        For this, we first iterate over the events 
+
+{% highlight python %}
+for event in events:
+    pass
+{% endhighlight %}
+
+        Make sure this is within the while loop. The iterator, i.e, the event object 
+        presented to us here is pretty useful. We can access it's <b>type</b> attribute 
+        which will be an integer, and check if that is going to be the type of event that 
+        corresponds to closing the window. 
+        Here's how it is done,
+
+{% highlight python %}
+for event in events:
+    if event.type == pygame.QUIT:
+        pass
+{% endhighlight %}
+
+        pygame.QUIT is the integer constant that represents trying to quit the application. 
+        Now, you can guess what to do next, we need to exit the loop. We ran this using a 
+
+{% highlight python %}
+while True:
+{% endhighlight %}
+
+        So the only way to exit this is to throw an exception, the way one of the 
+        lead contributors of the pygame project itself like to do it, and I have adopted as well, 
+        is to raise SystemExit. Alternatively you can import sys, and run sys.exit(). 
+
+{% highlight python %}
+while True:
+    events = pygame.event.get()
+
+    for event in events:
+        if event.type == pygame.QUIT:
+            raise SystemExit
+{% endhighlight %}
+        
+        And there we have it! Our game should close when we press the X button accordingly. 
+
+        As a rule of thumb, you should also fill your screen with a solid colour every frame, as to avoid 
+        seeing the last frame in the current one, and also updating your display. 
+
+{% highlight python %}
+screen.fill("black")
+
+pygame.display.flip()
+{% endhighlight %}
+
+        We can pass strings for colours in pygame, 
+        as covered in the <a href="https://pygame-guide-for-sneks.github.io/intro/colors">colors</a> article.
+
+        pygame.display.flip just updates what is shown on the window, it does the same as pygame.display.update but 
```

Comment:
You can enclose inline code with `<code></code>` tags. There's also a css file in the most up-to-date version of the repository that you cna include in your `<head>` to give inline code blocks some nice formatting
```
<head>
<link rel="stylesheet" href="../../styles/code_tags.css">
</head>
```

Reply:
For example, you could enclose pygame.display.flip like so
```
<code class="code">pygame.display.flip</code>
```